snap/snapcraft.yaml

```diff
@@ -40,6 +40,14 @@ apps:
       DAEMON_CONFIG_HOME: *daemon-config # temporary
       XTABLES_LIBDIR: $SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/xtables
     daemon: simple
+    plugs:
+      - kvm
+      - libvirt
+      - multipass-support
+      - network-bind
+      - network-control
+      - firewall-control
```

Comment:
So we need `kvm`, `network-control` and `firewall-control` autoconnected. `network-bind` [already is](https://snapcraft.io/docs/network-bind-interface), and we just got `multipass-support`, too.

Reply:
> `<jdstrand> done`